suite-native/module-home/src/screens/HomeScreen/components/EnableViewOnlyBottomSheet.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import {
+    selectDevice,
+    selectIsDeviceAuthorized,
+    selectIsPortfolioTrackerDevice,
+    selectIsDeviceInViewOnlyMode,
+    toggleRememberDevice,
+} from '@suite-common/wallet-core';
+import { BottomSheet, Box, Button, CenteredTitleHeader, VStack } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { useIsBiometricsInitialSetupFinished } from '@suite-native/biometrics';
+import { Translation } from '@suite-native/intl';
+import {
+    selectViewOnlyCancelationTimestamp,
+    setViewOnlyCancelationTimestamp,
+} from '@suite-native/module-settings';
+
+import { DisconnectedTrezorSvg } from '../../../assets/DisconnectedTrezorSvg';
+
+const SHOW_TIMEOUT = 1500;
+
+const svgStyle = prepareNativeStyle(utils => ({
+    paddingTop: utils.spacings.small,
+    width: '100%',
+    height: 206,
+}));
+
+const buttonWrapperStyle = prepareNativeStyle(utils => ({
+    gap: utils.spacings.medium,
+}));
+
+export const EnableViewOnlyBottomSheet = () => {
+    const dispatch = useDispatch();
+    const { applyStyle } = useNativeStyles();
+    const { isBiometricsInitialSetupFinished } = useIsBiometricsInitialSetupFinished();
+    const isDeviceAuthorized = useSelector(selectIsDeviceAuthorized);
+    const device = useSelector(selectDevice);
+    const isPortfolioTrackerDevice = useSelector(selectIsPortfolioTrackerDevice);
+    const viewOnlyCancelationTimestamp = useSelector(selectViewOnlyCancelationTimestamp);
+    const isDeviceInViewOnlyMode = useSelector(selectIsDeviceInViewOnlyMode);
+
+    const [isVisible, setIsVisible] = useState(false);
+
+    // show the bottom sheet if:
+    //     does not have view only mode enabled already
+    //     the device is authorized
+    //     not a portfolio tracker
+    //     the user hasn't made a choice yet
+    //     and biometrics initial setup was decided
+
+    const canBeShowed =
```

Comment:
feature flag has to be check here, right?

Reply:
1dadf9384b